latch_cli/docker_utils/__init__.py

```diff
@@ -93,6 +93,7 @@ def get_epilogue(wf_type: WorkflowType = WorkflowType.latchbiosdk) -> List[str]:
             "copy .latch/bin/nextflow /root/nextflow",
             "copy .latch/.nextflow /root/.nextflow",
             "copy .latch/nf_entrypoint.py /root/nf_entrypoint.py",
+            "run apt-get update && apt-get install -y default-jre-headless",
```

Comment:
maybe move this above
`f.write("copy . /root/\n\n")` 
so that the we don't have to re-run this command everytime the we re-build the dockerfile>

Reply:
+1